cfgov/data_research/forms.py

```diff
@@ -1,18 +1,112 @@
+from __future__ import absolute_import, unicode_literals
+
 from django import forms
 
-from .models import ConferenceRegistration
+from core.govdelivery import get_govdelivery_api
+from data_research.models import ConferenceRegistration
+from data_research.widgets import (
+    CheckboxSelectMultiple, EmailInput, TextInput, Textarea
+)
+
+
+class ConferenceRegistrationForm(forms.Form):
+    """Form for registering an attendee to a conference.
+
+    Creates a ConferenceRegistration model instance upon successful
+    registration, and also subscribes the registrant to a GovDelivery
+    mailing list.
 
+    If save(commit=False) is used, a model instance is created but not
+    persisted to the database, and GovDelivery subscription is skipped.
+    """
+    SESSIONS = tuple((s, s) for s in (
+        'Thursday morning',
+        'Thursday lunch',
+        'Thursday afternoon',
+        'Friday morning',
+    ))
 
-class ConferenceRegistrationForm(forms.ModelForm):
-    class Meta:
-        model = ConferenceRegistration
-        fields = '__all__'
+    DIETARY_RESTRICTIONS = tuple((dr, dr) for dr in (
```

Comment:
Would it make sense to separate this content out, so that if someone wants to update it, it doesn't have to go through a deploy?

Reply:
This is an excellent question. In the 2016 iteration, it was more this way, where the various options were configurable via Wagtail (although the fields weren't -- those were hardcoded into the model, whereas I've gone the other way here). In practice, though, since we're launching the form very soon and I'm implementing all of the changes and content, there's no real advantage to making it configurable there. It also adds more complexity to the code to pass it through.